upgrading/upgrade-ui/1_codegen.md

```diff
@@ -0,0 +1,52 @@
+We start with the following setup: Your social network of `create-daml-app-0.1.0` and `forum-0.1.0`
+are running and your users have been happily posting messages in the forum. You decide to upgrade to
+`create-daml-app-0.1.1` and hence you write a migration package `migration-0.1.0` and deploy it
+together with the new packages `create-daml-app-0.1.1`, `forum-0.1.1`.
+
+Now you want to drive the upgrade process of the `create-daml-app-0.1.0` DAML application from the
+UI.  If a user wants to login, whose `User` contract is still of the old `create-daml-app-0.1.0`
+package, you ask her kindly to upgrade to the newer package. 
+
+In order to do so, you need the generated JavaScript bindings for the newly available packages
+available. **Wait for the sandbox to boot in the background**. Then create them with the DAML
+Assistant like so:
+
+```
+daml codegen js create-daml-app-0.1.0.dar -o daml.js
+daml codegen js create-daml-app-0.1.1.dar -o daml.js
+daml codegen js forum-0.1.0.dar -o daml.js
+daml codegen js migration-0.1.0.dar -o daml.js
+```{{execute T1}}
+
+Once finished, open `ui/package.json`{{open}} in the IDE and add the new dependencies of
+`create-daml-app-0.1.1`, `forum-0.1.0` and `migration-0.1.0`
+
+<pre class="file" data-target="clipboard">
+  "dependencies": {
+    "@daml.js/create-daml-app-0.1.0": "file:../daml.js/create-daml-app-0.1.0",
+    "@daml.js/create-daml-app": "file:../daml.js/create-daml-app-0.1.1",
+    "@daml.js/forum-0.1.0": "file:../daml.js/forum-0.1.0",
+    "@daml.js/migration-v0-v1": "file:../daml.js/migration-0.1.0",
+    "@daml/ledger": "1.1.1",
+    "@daml/react": "1.1.1",
+    "@daml/types": "1.1.1",
+    "jwt-simple": "^0.5.6",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "dotenv": "^8.2.0",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.88.1"
+  }
+</pre>
+
+and install them with
+
+```
+cd ui
+yarn install
+```{{execute T1}}
+
+Notice that we renamed the `create-daml-app-0.1.0` import from `create-daml-app` to
+`create-daml-app-0.1.0`, while we import `create-daml-app-0.1.1` now as `create-daml-app`. This
```

Comment:
```suggestion
Notice that `create-daml-app` dependency is now pointing to `create-daml-app-0.1.1`. We point to the old old contracts and its bindings explicitly via `create-daml-app-0.1.0` dependency.
```
Had to read the text couple of times, the above suggestion made sense to me.

Reply:
I rephrased it a little.